src/modules/flow/oauth/oauth.c

```diff
@@ -50,30 +50,29 @@ struct v1_request_data {
     char *timestamp, *nonce, *callback_url;
 };
 
-struct oauth_node_type {
-    struct sol_flow_node_type base;
+static struct {
     struct sol_http_server *server;
-    uint16_t server_ref;
-};
+    uint16_t ref;
+} oauth;
 
 static int
-server_ref(struct oauth_node_type *oauth)
+ref(void)
```

Comment:
[minor] I would continue with the `server_ref()` name, at least to maintain consistency with `server_unref()`